docs/content/integrations/airlift/tutorial.mdx

```diff
@@ -1 +1,1258 @@
+
+```bash
+gh repo clone dagster-io/airlift-tutorial
+cd airlift-tutorial
+```
+
+First we strongly recommend that you setup a fresh virtual environment and that you use `uv`.
+
+```bash
+pip install uv
+uv venv
+source .venv/bin/activate
+```
+
+## Running Airflow locally
+
+The tutorial example involves running a local Airflow instance. This can be done by running the following commands from the root of the `tutorial-example` directory.
+
+First, install the required python packages:
+
+```bash
+make airflow_install
+```
+
+Next, scaffold the Airflow instance, and initialize the dbt project:
+
+```bash
+make airflow_setup
+```
+
+Finally, run the Airflow instance with environment variables set:
+
+```bash
+make airflow_run
+```
+
+This will run the Airflow Web UI in a shell. You should now be able to access the Airflow UI at <http://localhost:8080>, with the default username and password set to `admin`.
+
+You should be able to see the `rebuild_customers_list` DAG in the Airflow UI, made up of three tasks: `load_raw_customers`, `run_dbt_model`, and `export_customers`.
+
+## Peering Dagster to your Airflow instance
+
+The first step is to peer your Airflow instance with a Dagster code location, which will create an asset representation of each of your Airflow DAGs that you can view in Dagster. This process does not require any changes to your Airflow instance.
+
+First, you will want a new shell and navigate to the same directory. You will need to set up the `dagster-airlift` package in your Dagster environment:
+
+```bash
+source .venv/bin/activate
+uv pip install 'dagster-airlift[core]' dagster-webserver dagster
+```
+
+Next, create a `Definitions` object using `build_defs_from_airflow_instance`. You can use the empty [`tutorial_example/dagster_defs/definitions.py`](./tutorial_example/dagster_defs/definitions.py) file as a starting point:
+
+```python file=../../experimental/dagster-airlift/examples/tutorial-example/tutorial_example/dagster_defs/stages/peer.py
+from dagster_airlift.core import (
+    AirflowInstance,
+    BasicAirflowAuthBackend,
+    build_defs_from_airflow_instance,
+)
+
+defs = build_defs_from_airflow_instance(
+    airflow_instance=AirflowInstance(
+        # other backends available (e.g. MwaaSessionAuthBackend)
+        auth_backend=BasicAirflowAuthBackend(
+            webserver_url="http://localhost:8080",
+            username="admin",
+            password="admin",
+        ),
+        name="airflow_instance_one",
+    )
+)
+```
+
+This function creates:
+
+- An external asset representing each DAG. This asset is marked as materialized whenever a DAG run completes.
+- A sensor that polls the Airflow instance for operational information. This sensor is responsible for creating materializations when a DAG executes. The sensor must remain on in order to properly update execution status.
+
+Let's set up some environment variables, and then point Dagster to see the asset created from our Airflow DAG:
```

Comment:
This sentence lost me a bit, I don't recall seeing the environment variables in the above code, so we may need to explain it a bit more.

I think the sentence:

> and then point Dagster to see the asset created from our Airflow DAG

Could be rephrased as something like, "Now that we've updated our `definitions.py` file, let's run Dagster and see the peered assets." Or something along those lines.